lib/suspenders/generators/app_generator.rb

```diff
@@ -48,6 +48,7 @@ def suspenders_customization
       invoke :setup_dotfiles
       invoke :setup_git
       invoke :setup_database
+      invoke :setup_app_json
```

Comment:
Should this happen only when the heroku flag is on? Should it happen before `create_heroku_pipeline`?